demo/api/src/db/migrations/Migration20240619092554.ts

```diff
@@ -0,0 +1,7 @@
+import { Migration } from "@mikro-orm/migrations";
+
+export class Migration20240619092554 extends Migration {
```

Comment:
Right now all migrations are in the demo - there is already a separate ticket to move the migrations into the module. I haven't figured out a way to combine the migrations with the comet migrations yet and to generate the columns for the scope and attributes dynamically. But this will be handled in upcoming ticket/PR - as soon as i have time for it

Reply:
As of today: yes, the migrations need to be done directly in the project unfortunately.